content/wasm-languages/webassembly-language-support.md

```diff
@@ -39,7 +39,7 @@ Some languages, like CSS, PowerShell, and "Shell", don't really have a meaningfu
 | [Kotlin (JVM)][Kotlin]    | ✅    | ✅      | ✅   | ⏳       |
 | [Kotlin (Wasm)][Kotlin]   | ⏳    | ✅      | ✅   | ❌       |
 | [Rust][Rust]              | ✅    | ✅      | ✅   | ✅       |
-| [Dart][Dart]              | ❌    | ⏳      | ❌   | ❌       |
+| [Dart][Dart]              | ✅    | ✅      | ⏳   | ❌       |
 
 * _Core_ means there is an implementation of WebAssembly 1.0
```

Comment:
I'm curious about this definition. GC languages require 2.0, right? So should `Core` column be ❌ for Dart, Java, Kotlin, etc?

If not, then I believe Dart should be ✅

Reply:
@technosophos I'm going to merge this on the basis of Kotlin being green, but I may be misunderstanding your intent with the Core column.  If so let me know and we can put it back to what you intended (running it past Kyle of course).